src/astronomer/flask_appbuilder/security.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2019 Astronomer Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from flask import request, abort
+from flask_login import login_user, current_user
+from flask_appbuilder.security.manager import AUTH_REMOTE_USER
+from flask_appbuilder.security.views import AuthView
+try:
+    from airflow.www_rbac.security import AirflowSecurityManager, EXISTING_ROLES
+except ImportError:
+    from airflow.www.security import AirflowSecurityManager, EXISTING_ROLES
+from logging import getLogger
+from jwcrypto import jwt, jwk, jws
+import json
+log = getLogger(__name__)
+
+
+class AstroSecurityManagerMixin(object):
```

Comment:
not sure that we need python2 here, better to remove `object`

Reply:
@schnie What versions of Python do we support? Having Py2 support is the difference between `(object):` and `:` here (though I haven't tested on Py2.

Reply:
if py2 is a case I'm okey with object. wo `object` it's new style classes https://wiki.python.org/moin/NewClassVsClassicClass

Reply:
So far we've been only supporting python 3, since py2 is about to die out officially. I don't think we've had too much push back on that. @vparekh94 how many people are asking about py2?

Reply:
we haven't had any customers ask about python2 in a while - I think if someone absolutely _needs_ it, it  might be something to handle w/ services. @ryw what do you think